superset-frontend/cypress-base/cypress/integration/dashboard/drilltodetail.test.ts

```diff
@@ -58,220 +58,243 @@ function closeModal() {
   });
 }
 
+function setTopLevelTab(tabName: string) {
+  cy.get("div#TABS-TOP div[role='tab']").contains(tabName).click();
+}
+
 describe('Drill to detail modal', () => {
   before(() => {
-    cy.visit(ECHARTS_DASHBOARD);
-    ECHARTS_CHARTS.forEach(waitForChartLoad);
```

Comment:
I think we should wait for the charts that should be rendered to load, otherwise we might cause some flakiness if any chart has delay